docroot/profiles/custom/sitenow/config/sync/config_ignore.settings.yml

```diff
@@ -7,8 +7,6 @@ ignored_config_entities:
   10: uiowa_footer.settings
   12: views.view.articles
   14: 'webform.webform.*'
-  16: 'migrate_plus.migration.*'
-  18: migrate_plus.migration_group.sitenow_migrate
```

Comment:
Do these need to be ignored? If so, they need to be ignored in the split config since #743.

Reply:
that would be ideal to prevent commits